sdcm/nemesis.py

```diff
@@ -2971,6 +3030,17 @@ def disrupt(self):
         time.sleep(300)
 
 
+class AddRemoveDcNemesis(Nemesis):
+
+    disruptive = False
```

Comment:
@roydahan , isn't running `nodetool repair` kind of a problem (for parallel nemesis for example)?
if so, should it be considered as `disruptive = True`?

Reply:
No.
Why would repair considered as disruptive?

Reply:
it will not be disruptive per se, but in terms of functionality, it cannot be run in parallel with most of other disruptive ones (thinking about parallel nemesis) as it will fail, and also might conflict with other `nodetool` commands